Deprecated --cui-bg-accent-strong-* and --cui-z-index-toast design tokens (#3715)
sumup-oss/circuit-ui: .changeset/silent-snails-shave.md

---
"@sumup-oss/stylelint-plugin-circuit-ui": patch
---

> Updated the `no-deprecated-custom-properties` rule to support deprecated tokens that have no direct replacement. A guidance message when an auto-fixable replacement is not available can now be provided.

Reviewer: ```suggestion
Updated the `no-deprecated-custom-properties` rule to support deprecated tokens that have no direct replacement. The rule now requires an additional message for deprecated tokens with no direct replacement.
```
